functions/Remove-DbaDatabaseSafely.ps1

```diff
@@ -149,7 +149,8 @@ If there is a DBCC Error it will continue to perform rest of the actions and wil
 		[object[]]$Database,
 		[parameter(Mandatory = $false)]
 		[object]$Destination = $sqlinstance,
```

Comment:
This should be set to `[DbaInstanceParameter]` as well...while you are updating.